### PR TITLE
Add Travis configured.

### PR DESCRIPTION
**Pull request by Maninda**
## Purpose
Try to fix Travis CI

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,25 +13,32 @@
...
+  # - echo -e "\033[36m${BALLERINA_VERSION}\033[m"
 
-  - DISTRIBUTION_NAME=ballerina-${BALLERINA_VERSION}
-  - echo -e "\033[36m${DISTRIBUTION_NAME}\033[m"
+  # - DISTRIBUTION_NAME=ballerina-${BALLERINA_VERSION}
+  # - echo -e "\033[36m${DISTRIBUTION_NAME}\033[m"
 
-  - DOWNLOAD_CHANNEL=${CHANNEL:-nightly}
-  - echo -e "\033[36m${DOWNLOAD_CHANNEL}\033[m"
+  # - DOWNLOAD_CHANNEL=${CHANNEL:-nightly}
```
Comment (Asitha) [inline .travis.yml]: Why have we commented this out?

File: .travis.yml [hunk 2]
```diff
@@ -13,25 +13,32 @@
...
 
-  - ballerina --version
+  # - ballerina --version
+
+  # - ballerina build --skip-tests amazonsqs
 
-  - ballerina build --skip-tests amazonsqs
\ No newline at end of file
+
+  - wget https://product-dist.ballerina.io/downloads/1.0.0/ballerina-linux-installer-x64-1.0.0.deb
```
Comment (Asitha) [inline .travis.yml]: Can't we use the parameterized variables for this?

File: .travis.yml (hunk 1 above)
Comment (Maninda) [reply .travis.yml]: This commit is just a test to fix Trvis CI. Will fix the format and comments in next PR.

File: .travis.yml (hunk 2 above)
Comment (Maninda) [reply .travis.yml]: We have not yet parameterized variables as Ballerina team has not confirmed the hosted location and name format of the distribution package.

**Pull request merged**